recipe/meta.yaml

```
@@ -21,7 +23,7 @@ requirements:
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - setuptools_scm
-    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
```

I suspect you need both
```suggestion
    - {{ compiler('c') }}
    - {{ stdlib("c") }}
```